app/iot/devices.py

```diff
@@ -1,64 +1,79 @@
-import time
+import asyncio
 
 from .message import MessageType
```

Comment:
Ensure that `MessageType` is correctly defined in the `message` module. If it's not defined or imported correctly, it will cause an error when trying to use it in the `send_message` methods.